lib/note-revisions/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import NotePreview from '../components/note-preview';
+import TagChip from '../components/tag-chip';
+import { noteCanonicalTags } from '../state/selectors';
+import { tagHashOf } from '../utils/tag-hash';
+
+import type * as S from '../state';
+import type * as T from '../types';
+
+type OwnProps = {
+  noteId: T.EntityId;
+  note?: T.Note;
+};
+
+type StateProps = {
+  tags: Array<{ name: T.TagName; deleted: boolean }>;
+  noteId: T.EntityId | null;
+  note: T.Note | null;
+};
+
+type Props = OwnProps &
+  StateProps &
+  Pick<React.HTMLProps<HTMLDivElement>, 'aria-hidden'>;
+
+export class NoteRevisions extends Component<Props> {
+  static displayName = 'NoteRevisions';
+
+  render() {
+    const { note, noteId, tags, 'aria-hidden': ariaHidden } = this.props;
+
+    return (
+      <div aria-hidden={ariaHidden} className="note-revisions">
+        <NotePreview noteId={noteId} note={note} />
+        <div className="tags">
```

Comment:
can we double-check the style here with `tags` as a bare value. I'm not as familiar with the rest of the CSS in this app but I do know we generally like to avoid such generic names as well as heavy nesting in SCSS (again I say I can't remember off-hand what our norms are in this repo).

might be valuable in any case to leave a more specific name here since `tags` is everywhere. something like `note-tag-list` possibly?

Reply:
Oh, good catch, I do agree that this class name is too generic. I've just renamed it in [this commit](https://github.com/Automattic/simplenote-electron/pull/2817/commits/f8fe9306c202cd62583b5ff62553a23db923369a), thanks!